hi-ml-histopathology/environment.yml

```diff
@@ -29,8 +29,7 @@ dependencies:
       - ruamel.yaml==0.16.12
       - tensorboard==2.6.0
       # Histopathology requirements
-      - cucim==22.2.0
-      - medmnist==2.0.2
+      - cucim==22.04.00
```

Comment:
in the PR description it says upgrading cucim but here we're downgrading - what was the need?

Reply:
Fernando mentioned that there was a bug fixed in the [latest version](https://github.com/rapidsai/cucim/pull/248). [Here's the release notes](https://github.com/rapidsai/cucim/releases) we're upgrading from 22.02.00 -> 22.04.00.